src/API/Management/Users.php

```diff
@@ -6,70 +6,126 @@
 
 class Users extends GenericResource
 {
-    public function get($user_id)
+    /**
+     * @param string $userId
+     *
+     * @return array
+     */
+    public function get($userId)
     {
-        $response = $this->httpClient->get(sprintf('/users/%s', $user_id));
+        $response = $this->httpClient->get(sprintf('/users/%s', $userId));
 
         return ResponseMediator::getContent($response);
     }
 
-    public function update($user_id, $data)
+    /**
+     * @param string $userId
+     * @param array $data
+     *
+     * @return array
+     */
+    public function update($userId, array $data)
     {
-        $response = $this->httpClient->patch(sprintf('/users/%s', $user_id), [], json_encode($data));
+        $response = $this->httpClient->patch(sprintf('/users/%s', $userId), [], json_encode($data));
 
         return ResponseMediator::getContent($response);
     }
 
-    public function create($data)
+    /**
+     * @param array $data
+     *
+     * @return array
+     */
+    public function create(array $data)
     {
         $response = $this->httpClient->post('/users', [], json_encode($data));
 
         return ResponseMediator::getContent($response);
     }
 
-    public function getAll($params = [])
+    /**
+     * @param array $params
+     *
+     * @return array
+     */
+    public function getAll(array $params = [])
     {
         return $this->search($params);
     }
 
-    public function search($params = [])
+    /**
+     * @param array $params
+     *
+     * @return array
+     */
+    public function search(array $params = [])
     {
         $response = $this->httpClient->get('/users?'.http_build_query($params));
 
         return ResponseMediator::getContent($response);
     }
 
-    public function delete($user_id)
+    /**
+     * @param string $userId
+     *
+     * @return array
+     */
+    public function delete($userId)
     {
-        $response = $this->httpClient->delete(sprintf('/users/%s', $user_id));
+        $response = $this->httpClient->delete(sprintf('/users/%s', $userId));
 
         return ResponseMediator::getContent($response);
     }
 
-    public function linkAccount($user_id, $post_identities_body)
+    /**
+     * @param string $userId
+     * @param string $postIdentitiesBody
```

Comment:
It should. I read the docs. https://auth0.com/docs/api/management/v2#!/Users/post_identities

The PR is updated.